docker/docker_files/Dockerfile

```diff
@@ -12,7 +12,7 @@ COPY setup.py /usr/src/
 RUN pip install --no-cache-dir more_itertools pytz sqlalchemy cachew \
                 appdirs urlextract python-magic \
                 tzlocal hug \
```

Comment:
could you remove `hug` as well please?

Reply:
Imade the changes, hope I did it right... I'm not that familiar with how github pull requests work (yet)